src/main/java/org/uwdigi/rag/config/ModelConfig.java

```diff
@@ -100,4 +125,26 @@ public void setLocalAiBaseUrl(String localAiBaseUrl) {
   public void setLocalAiModelName(String localAiModelName) {
     this.localAiModelName = localAiModelName;
   }
+
+  public ChatLanguageModel getSqlGenerationModel() {
```

Comment:
We remove this and let the user continue selecting the model for generating the SQL.